src/main/java/org/cru/godtools/api/translations/GodToolsTranslation.java

```diff
@@ -75,6 +77,19 @@ public static GodToolsTranslation assembleFromComponents(Package gtPackage,
 		return godToolsTranslation;
 	}
 
+	public Optional<PageStructure> getPage(String pageName)
+	{
+		for(PageStructure pageStructure : pageStructureList)
+		{
+			if(Objects.equal(pageStructure.getFilename(), pageName))
+			{
+				return Optional.fromNullable(pageStructure);
```

Comment:
Also good job of using Optional, I want to use it more.  It makes code much more readable.